about.html

```diff
@@ -1,14 +1,13 @@
 ---
 layout: page
 title: "About"
-description: "This is what I do."
+description: "Get to know GitHub Teacher"
 header-img: "img/about-bg.jpg"
 ---
 
-<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Saepe nostrum ullam eveniet pariatur voluptates odit, fuga atque ea nobis sit soluta odio, adipisci quas excepturi maxime quae totam ducimus consectetur?</p>
+<p>Thank you for visiting my blog. I am the teacher of all things GitHub and this blog is a place I can share my random learnings and connect with you, the GitHub community.</p>
 
-<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius praesentium recusandae illo eaque architecto error, repellendus iusto reprehenderit, doloribus, minus sunt. Numquam at quae voluptatum in officia voluptas voluptatibus, minus!</p>
-
-<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nostrum molestiae debitis nobis, quod sapiente qui voluptatum, placeat magni repudiandae accusantium fugit quas labore non rerum possimus, corrupti enim modi! Et.</p>
-
-<a href="htp:/broken.z"> broken link </a>
+<p>If you are interested in learning how to use Git and GitHub, check out these resources:</p>
+<ul>
+  <li><a href="htt://services.github.com/on-demand">GitHub's On-Demand Training Classes</a></li>
```

Comment:
This link doesn't look correct. Can you please change it to `http://services.github.com/on-demand`?